source/change-streams.rst

```diff
@@ -339,19 +421,54 @@ Single Server Topologies
 
 Presently, change streams cannot be initiated on single server topologies as they do not have an oplog.  Drivers MUST NOT throw an exception in this scenario, but instead rely on an error returned from the server.  This allows for the server to seamlessly introduce support for this in the future, without need to make changes in driver code.
 
+startAtClusterTime
+^^^^^^^^^^^^^^^^^^
+
+:since: 4.0
+
+``startAtClusterTime`` specifies that a ``changeStream`` will only return changes that occurred after the specified ``Timestamp``.
```

Comment:
"that occurred at or after ..."  as per Charlie's clarification in [SERVER-34818](https://jira.mongodb.org/browse/SERVER-34818)

Reply:
done